learn/tooling-guide/ballerina-shell.md

```diff
@@ -2,11 +2,213 @@
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Ballerina Shell
 permalink: /learn/tooling-guide/ballerina-shell/
+description: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 active: ballerina-shell
-intro: The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger. Thus, you can run or debug your Ballerina programs easily via the VS Code Ballerina extension by launching its debugger. 
+intro: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 redirect_from:
   - /learn/tooling-guide/ballerina-shell
 ---
 
+It allows you to evaluate snippets of code without having to write complete programs. 
+
+This will particularly be your interest if you are looking to try out the language and its new features, and also for debugging purposes.
+
+## Setting up the Prerequisites
+
+The Ballerina Shell is a part of the Ballerina distribution. It was first shipped with Ballerina Swan Lake Alpha1. Therefore, if you are using Swan Lake Alpha1 or a later version, you are all set. If not, for information on installing the latest version of Ballerina, see [downloads](https://ballerina.io/downloads/).
+
+## Starting the Shell
+
+Execute the command below to start Ballerina Shell.
+
+```bash
+bal shell
+```
+You view the putput below.
+
+```bash
+ 
+Welcome to Ballerina Shell REPL.
+Type /exit to exit and /help to list available commands.
+
+= 
+```
+
+## Stopping the Shell
+
+Execute the `/exit` command below in a new prompt to exit the Shell.
+
+```bash
+= /exit  
+```
+
+You view the putput below.
+
+```bash
+| Bye!!!
```

Comment:
Don't break any of the code segments like this. Include them exactly like they are in the doc

Reply:
Fixed.